patches/v8/dcheck.patch

```diff
@@ -18,20 +18,3 @@ index 0b6bdbc0328dfc979531dbbb25c5fc7b02f661bd..191ee0bb424471c1240835dedc7f25b8
    i::Isolate* i_isolate = reinterpret_cast<i::Isolate*>(this);
    i_isolate->default_microtask_queue()->PerformCheckpoint(this);
  }
-diff --git a/src/heap/heap.cc b/src/heap/heap.cc
-index 41f2378ba770a96d9722b8d50d28f3562ebf5101..59338d1d293d5325e89052dd226a9ad4f9e8b46a 100644
---- a/src/heap/heap.cc
-+++ b/src/heap/heap.cc
-@@ -6165,9 +6165,9 @@ void Heap::TearDown() {
```

Comment:
Patch was originally added in https://github.com/electron/libchromiumcontent/pull/464 . This is an attempt to see if it is still required.